jobs/deploy-webapp.groovy

```diff
@@ -372,79 +372,85 @@ def mergeFromMasterAndInitializeGlobals() {
          }
       }
 
-      withVirtualenv.python3() {
-         dir("webapp") {
-            clean(params.CLEAN);
+      dir("webapp") {
+         clean(params.CLEAN);
+      }
+   }
+}
 
-            // Let's do a sanity check.
-            def headSHA1 = exec.outputOf(["git", "rev-parse", "HEAD"]);
-            if (params.GIT_REVISION != headSHA1) {
-               notify.fail("Internal error: " +
-                           "HEAD does not point to the deploy-branch");
-            }
+def initializeGlobals() {
```

Comment:
This will be easier to read if you tell git to ignore whitespace (via the gear).